packages/atlas-service/src/main.ts

```diff
@@ -515,6 +528,9 @@ export class AtlasService {
   }
 
   static async signOut(): Promise<void> {
+    if (!this.currentUser) {
+      throw new Error("Can't sign out if not signed in yet");
+    }
```

Comment:
Hmmm, good point. My thinking here was that this is an unexpected behavior (like literally there should be no way to call sign out in UI if `currentUser` is not set) in the app and so if this somehow happens, we should indicate it in the implementation by throwing so it can be caught during development. Am I overthinking this? What do you think?